utils/validation/scripts/validation.py

```diff
@@ -0,0 +1,19 @@
+import yamale
+from git import Repo
+import re
+
+try:
+    experiment_schema = yamale.make_schema("utils/validation/schemas/experiment.yaml")
+    repo = Repo(".")
+    files = repo.git.execute(["git", "diff", "--name-only", "main"]).split("\n")
+    experiments_re = re.compile(r"^experiments\/.+")
+    experiments_files = list(
+        filter(lambda filename: bool(experiments_re.match(filename)), files)
+    )
+    for experiment in experiments_files:
```

Comment:
So this validates all the files present in the experiment folder using `experiments.yaml` for the validation ?